addons/hw_drivers/iot_handlers/drivers/DisplayDriver_L.py

```diff
@@ -232,6 +251,7 @@ def display(self, display_identifier=None):
             'display_ifaces': display_ifaces,
             'display_identifier': display_identifier,
             'pairing_code': connection_manager.pairing_code,
+            'hostname': socket.gethostname(),
```

Comment:
as you refactor a lot of things already, wouldn't it be worth to create a helper function for `socket.gethostname()` ? Pretty sure it could be cached with no major downsides (apart maybe on windows ?)